i18n/en.pot

```diff
@@ -5,8 +5,8 @@ msgstr ""
 "Content-Type: text/plain; charset=utf-8\n"
 "Content-Transfer-Encoding: 8bit\n"
 "Plural-Forms: nplurals=2; plural=(n != 1)\n"
-"POT-Creation-Date: 2022-12-09T22:44:29.637Z\n"
-"PO-Revision-Date: 2022-12-09T22:44:29.637Z\n"
+"POT-Creation-Date: 2022-12-16T07:59:16.298Z\n"
+"PO-Revision-Date: 2022-12-16T07:59:16.298Z\n"
 
```

Comment:
We have also `i18n/es.po`, which is not updated. To do so, run `yarn localize`

Reply:
Thanks @tokland , got this. recent commits on latest branches now have yarn localize builds included in the commits.